examples/email-auth/src/pages/index.tsx

```diff
@@ -43,75 +39,57 @@ export default function AuthPage() {
   } = useForm<InjectCredentialsFormData>();
 
   const auth = async (data: AuthFormData) => {
-    if (iframeStamper === null) {
+    if (authIframeClient === null) {
       throw new Error("cannot initialize auth without an iframe");
     }
 
     const response = await axios.post("/api/auth", {
       suborgID: data.suborgID,
       email: data.email,
-      targetPublicKey: iframeStamper.publicKey(),
+      targetPublicKey: authIframeClient!.iframePublicKey,
       invalidateExisting: data.invalidateExisting,
     });
 
     setAuthResponse(response.data);
   };
 
   const injectCredentials = async (data: InjectCredentialsFormData) => {
-    if (iframeStamper === null) {
-      throw new Error("iframeStamper is null");
+    if (authIframeClient === null) {
+      throw new Error("iframe client is null");
     }
     if (authResponse === null) {
       throw new Error("authResponse is null");
     }
-
     try {
-      await iframeStamper.injectCredentialBundle(data.authBundle);
+      await authIframeClient!.injectCredentialBundle(data.authBundle);
     } catch (e) {
       const msg = `error while injecting bundle: ${e}`;
       console.error(msg);
       alert(msg);
       return;
     }
 
-    const client = new TurnkeyClient(
-      {
-        baseUrl: process.env.NEXT_PUBLIC_BASE_URL!,
-      },
-      iframeStamper
-    );
-
     // get whoami for suborg
-    const whoamiResponse = await client.getWhoami({
+    const whoamiResponse = await authIframeClient!.getWhoami({
       organizationId: process.env.NEXT_PUBLIC_ORGANIZATION_ID!,
     });
 
     const orgID = whoamiResponse.organizationId;
 
-    const activityPoller = createActivityPoller({
-      client,
-      requestFn: client.createWallet,
-    });
-
-    const completedActivity = await activityPoller({
-      type: "ACTIVITY_TYPE_CREATE_WALLET",
-      timestampMs: String(Date.now()),
+    const createWalletResponse = await authIframeClient!.createWallet({
       organizationId: orgID,
-      parameters: {
-        walletName: data.walletName,
-        accounts: [
-          {
-            curve: "CURVE_SECP256K1",
-            pathFormat: "PATH_FORMAT_BIP32",
-            path: "m/44'/60'/0'/0/0",
-            addressFormat: "ADDRESS_FORMAT_ETHEREUM",
-          },
-        ],
-      },
+      walletName: data.walletName,
+      accounts: [
```

Comment:
tiny nit: we could pull this accounts config out of `sdk-browser` :)

Reply:
Would require installing a new package which may get confusing to users as to why we would require sdk-browser for this example? Perhaps it makes sense to export from react as well

Reply:
sorry, meant `sdk-server`. both server and browser offer the helpers: https://github.com/tkhq/sdk/blob/main/packages/sdk-server/src/turnkey-helpers.ts